walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,14 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
+    Greeter greeter = new Greeter();
+
+    String greeting = greeter.greet("   Ada   ");
+
+    // Whitespace should be trimmed
+    Assert.assertEquals("Hello Ada", greeting);
+  }
```

Comment:
Nit: Maybe add tests for a newline (\n) and preservation of spaces. For example, greeter.greet('Angel Guevara') should return 'Hello Angel Guevara';

Reply:
Oh, I did not make any more tests since I was just following the introductory walkthrough. :)